archlinuxcn/lceda-pro/lilac.yaml

```
@@ -0,0 +1,18 @@
+maintainers:
+  - github: taotieren
+
+build_prefix: extra-x86_64
+
+pre_build_script: |
+  update_pkgver_and_pkgrel(_G.newver)
+
+post_build_script: |
+  git_pkgbuild_commit()
+  update_aur_repo()
+
+update_on:
+  - source: htmlparser
+    url: "https://lceda.cn/page/download?src=index"
+    xpath: "/html/body/div[1]/div[1]/div[4]/div[2]/table/tbody/tr[3]/td[2]/div[1]/span[1]"
```

```suggestion
    xpath: "/html/body/div[1]/div[1]/div[4]/div[2]/table/tbody/tr[3]/td[2]/div[1]/span[1]"
```
    xpath: "//table/tbody/tr[3]/td[2]/div/span[1]"